changelog

```diff
@@ -1,3 +1,240 @@
+    * The color of a wormhole's link on the map is now also used when that wormhole is a part of the travel plan. (@RisingLeaf, @warp-core)
+    * Increased the amount of motion blur that occurs to the background while hyperspacing. This can be toggled in the settings under the "extended jump effects" preference. (@Koranir)
+    * The map can now be opened from the shipyard or outfitter. (@TomGoodIdea)
+    * Externally docked fighters are now drawn in the hail panel when hailing a carrier. (@TomGoodIdea)
+    * The auto-selection of available jobs when you accept one job now moves to the next available job for the destination instead of reseting back to the top of the list of jobs for that destination. (@flowers9)
+    * Added a preference to blink the map mission marker for missions with a deadline more quickly if you have fewer extra days to reach the destination compared to the minimum number of days necessary to reach it. The default behavior is to simply blink based on the number of days until the deadline. (@TomGoodIdea)
+    * The player will now be warned when taking off if they don't have enough cargo space for outfits that are marked as in cargo. (@flowers9, @warp-core)
+    * Buttons on the planet panel when you are landed no longer disappear if you don't have a flagship. (@flowers9)
+    * Mission destination reminder messages are now colored in a new green "info" message level. (@RisingLeaf)
+    * Interface bars can now be defined to draw from the top left corner instead of only drawing from the bottom right. (@warp-core)
+    * Added a confirmation dialog to demanding tribute from a planet, to prevent the player accidentally demanding tribute from a planet and ruining their reputation. (@UnorderedSigh)
+    * Non-hyperdrive jumps on the map travel path are now drawn with dashed lines instead of solid lines. (@flowars9)
+    * The save file load panel now scrolls when you use arrow keys to select your pilot and save. (@warp-core)
+    * Planet labels are now much less likely to overlap with one another or with other objects in the system. (@flowers9)
+    * The escort HUD display is now defined by the interfaces file so that it can be modified without recompiling. (@warp-core)
+    * Fullscreen can now be toggled in all panels. (@TomGoodIdea)
+    * The net change in energy and heat when all systems are active is now visible on the energy/heat table for ships when viewed in the outfitter and shipyard. (@mOctave)
+    * The shipyard and outfitter map keys now have a color for denoting systems with parked ships or stored outfits. (@flowers9)
+    * If a planet has a different government from the system it is in, that is now displayed on the planet's detail card in the map. (@TomGoodIdea)
+    * Created a new preference to change the format that dates are displayed in. (@a-random-lemurian)
+    * All applicable takeoff warnings are now displayed at the same time instead of only displaying one warning. (@TomGoodIdea)
```

Comment:
```suggestion
    * All applicable takeoff warnings are now displayed at the same time instead of only displaying one warning. (@TomGoodIdea, @quyykk)
```